build_rsas_from_etas.gs

```diff
@@ -0,0 +1,390 @@
+/*
+    Name:        WeDiscover - ETA To RSA Builder, Google Ads Script
+    Description: A script to build RSA ad copy from existing ETA ad
+                 copy.
+    License:     https://github.com/we-discover/public/blob/master/LICENSE
+    Version:     1.0.0
+    Released:    2021-12-13
+    Contact:     scripts@we-discover.com
+*/
+
+// EDIT ME -- Check for RSAs in Paused Ads, Ad Groups or Campaigns
+var checkPausedCampaigns = false;
+var checkPausedAdGroups = false;
+var includePausedRsas = false;
+var pullFromPausedEtas = false;
+
+// Script entrypoint
+function main(){
+  var executionContext = getExecutionContext();
+  var topLevelAccountName = AdsApp.currentAccount().getName();
+  var topLevelAccountId = AdsApp.currentAccount().getCustomerId();
+  var ssName = topLevelAccountName + " (" + topLevelAccountId + ") | ETA to RSA Builder";
+  
+  Logger.log("Making Google Sheet...");
+  var ss = SpreadsheetApp.create(ssName);
+  
+  // If MCC, run data process on a loop through all accounts
+  if (executionContext === 'manager_account') {
+    var accountIterator = AdsManagerApp.accounts().get();
+    while(accountIterator.hasNext()) {
+      AdsManagerApp.select(accountIterator.next());
+      
+      var accountName = AdsApp.currentAccount().getName();
+      var accountId = AdsApp.currentAccount().getCustomerId();
+      var accountNameAndId = accountName + " (" + accountId + ")";  
+      var successCount = 0;
+      
+      Logger.log("Looking in account " + accountNameAndId + " ...");
+      var assets = getAssets(accountName, accountId);
+      if (assets !== 'no_rsas') {
```

Comment:
Is this correct? `!==`

Reply:
Yeah
In Javascript `==` means 'check value is the same regardless of type', so `10 == "10"` is `true`
There's also `===` means means 'check same value and same type' so `10 === "10"` is `false`

`!=` and `!==` are the negative equivalents. In general you want to use the `===` and `!==` as they're stricter.

Reply:
https://www.w3schools.com/js/js_comparisons.asp